README.md

```diff
@@ -1,118 +1,120 @@
-
 1. Create a repository called `control-repo`
-
-1. * Create a user called `r10k` with a password of `puppet`.
-   * Make the r10k user an admin of the `PUP` project.
-
+1. Create a user called `r10k` with a password of `puppet`.
+    * Make the r10k user an admin of the `PUP` project.
 1. Either use the admin user to test pushing code, or create a user for yourself and add your SSH key to that user.
-  * If making a user for yourself, give your user account read/write or admin privilege to the `PUP` project.
-
+    * If making a user for yourself, give your user account read/write or admin privilege to the `PUP` project.
 1. Clone this control repository to your laptop/workstation
- - `git clone <repository url>`
- - `cd control-repo`
-
+    * `git clone <repository url>`
+    * `cd control-repo`
 1. Remove this repository as the origin remote
- - `git remote remove origin`
-
+    * `git remote remove origin`
 1. Add your internal repository as the origin remote
- - `git remote add origin <url of your bitbucket repository>`
-
+    * `git remote add origin <url of your bitbucket repository>`
 1. Push the production branch of the repository from your machine up to your git server
- - `git push origin production`
+    * `git push origin production`
 
 ### GitHub
 
 1. Prepare your local git client to authenticate with GitHub.com or a local GitHub Enterprise instance.
-  - https://help.github.com/articles/generating-ssh-keys/
-  - https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/
-
-2. Create a repository called `control-repo` in your user account or organization.  Ensure that "Initialize this repository with a README" is not selected.
-  - https://help.github.com/articles/creating-a-new-repository/
-
-3. Make a note of your repository URL (HTTPS or SSH, depending on your security configuration).
+    * <https://help.github.com/articles/generating-ssh-keys/>
+    * <https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/>
+1. Create a repository called `control-repo` in your user account or organization. Ensure that "Initialize this repository with a README" is not selected.
+    * <https://help.github.com/articles/creating-a-new-repository/>
+1. Make a note of your repository URL (HTTPS or SSH, depending on your security configuration).
+1. Clone this control repository to your laptop/workstation:
+    * `git clone <repository url>`
+    * `cd control-repo`
+1. Remove this repository as the origin remote:
+    * `git remote remove origin`
+1. Add your internal repository as the origin remote:
+    * `git remote add origin <url of your github repository>`
+1. Push the production branch of the repository from your machine up to your git server
+    * `git push origin production`
 
-4. Clone this control repository to your laptop/workstation:
-  - `git clone <repository url>`
-  - `cd control-repo`
+## Where Did All The Previous Code Go?
```

Comment:
Although it pains me to say so, I think we should probably remove this section altogether.  

The way it's worded makes the rampup-control-repo sound complicated instead of a simplification of the process of setting up code manager.  If we were to have a section here I think it should be more like:

# What about code manager?  

If you haven't installed code manager you may prefer to use the rampup-control-repo which is this control-repo plus a module and specific steps for installing PE and configuring code manager.

Reply:
@npwalker I agree and made some additional changes.